packages/starknet-snap/package.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "@consensys/starknet-snap",
-  "version": "2.4.0",
+  "version": "2.4.1",
```

Comment:
we dont need to chg version, when merge to main, it will auto upgrade

Reply:
Done